.editorconfig

```
@@ -4,23 +4,49 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = space
-indent_size = 2
 insert_final_newline = true
 max_line_length = 80
 trim_trailing_whitespace = true
 
-[*.yml]
+[*.{js,cjs}]
+indent_style = space
+indent_size = 2
```

Update to consistently order (alphabetically) as:

```suggestion
indent_size = 2
indent_style = space
```